### ISSUE TITLE
UCUM subset of Chinese defs from OM list 

### ISSUE THREAD

**Issue by kaiiam** [kaiiam/UO_revamp#15]
From @HajoRijgersberg [OM chinese labels list]( https://github.com/HajoRijgersberg/OM/tree/master/Chinese%20labels), filter to get those OM terms matching (as many as possible of) the 310 base UCUM terms for @zhengj2007 to review/fix. 

Later once the new system produces Chinese defs for derived units we can pass those to OM for it's use.

**Comment by HajoRijgersberg** [comment]
> Later once the new system produces Chinese defs for derived units we can pass those to OM for it's use.

Thanks Kai, if you can send them in some csv or Excel format with some identification for the intended units (in a way that's feasible for you), that would be great. Like a file with two columns:
- Unit identification (like the name or the symbol)
- Chinese def

Thanks in advance! Good luck!

**Comment by kaiiam** [comment]
Yes @HajoRijgersberg no worries I'll make an tabular export version once that's all ready with the OM purls/ID chinese labels and defs etc or whatever else you want from the new system. [Robot export](robot.obolibrary.org/export) is another awesome tool from James which can do this easily enough, plus our mappings Simon made to OM.
